src/routes.rs

```diff
@@ -97,57 +96,42 @@ pub async fn get_content_no_auth(
     }
 }
 
-#[post("/<_orbit_id>", format = "multipart/form-data", data = "<batch>")]
-pub async fn batch_put_content(
-    _orbit_id: CidWrap,
-    orbit: PutAuthWrapper,
-    batch: Form<Vec<PutContent>>,
-) -> Result<String, (Status, &'static str)> {
-    let mut uris = Vec::<String>::new();
-    for content in batch.into_inner().into_iter() {
-        uris.push(
-            orbit
-                .0
-                .put(&content.content, content.codec)
-                .await
-                .map_or("".into(), |cid| {
-                    orbit.0.make_uri(&cid).map_or("".into(), |s| s)
-                }),
-        );
-    }
-    Ok(uris.join("\n"))
-}
-
-#[post("/<_orbit_id>", data = "<data>", rank = 2)]
+#[post("/<_orbit_id>", data = "<data>", rank = 1)]
 pub async fn put_content(
     _orbit_id: CidWrap,
-    orbit: PutAuthWrapper,
     data: Data,
     codec: SupportedCodecs,
-) -> Result<String, (Status, &'static str)> {
+    orbit: PutAuthWrapper,
```

Comment:
I realise we never check the orbit against the orbit ID in the URL

Reply:
this has now been implemented in the request guards for the `*AuthWrapper`s